build.rs

```diff
@@ -88,4 +87,8 @@ impl Version {
         let nightly = channel.map_or(false, |c| c == "dev" || c == "nightly");
         Ok(Self { minor, nightly })
     }
+
+    fn print(&self) -> String {
+        format!("Version {{ minor: {}, nightly: {} }}\n", self.minor, self.nightly)
```

Comment:
I don't think `Display` format is also guaranteed by Rust, but I don't think `u32` and `bool`'s `Display` (`ToString`) format will be invalid literal.